lib/oneview-sdk/resource/api500/synergy/storage_system.rb

```diff
@@ -9,7 +9,7 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-require_relative '../../api300/synergy/storage_system'
+require_relative '../../api500/c7000/storage_system'
```

Comment:
We could back just one folder level to reference the file: `require_relative '../c7000/storage_system'`

Reply:
Yes, you're right :)